components/frontend/src/metric/MetricDetails.test.jsx

```diff
@@ -120,48 +130,6 @@ async function renderMetricDetails(stopFilteringAndSorting, connection_error, fa
     return result
 }
 
-beforeEach(() => {
-    history.push("")
-    fetch_server_api.fetch_server_api.mockImplementation(() => Promise.resolve())
-})
-
-afterEach(() => {
-    vi.clearAllMocks()
-})
-
-it("switches tabs", async () => {
```

Comment:
Are these deleted tests tested elsewhere now?

Reply:
No, I removed them because they were basically testing MUI (switching tabs) instead of our own code.